### DISCUSSION TITLE
Spider

### DISCUSSION THREAD

**Discussion by deftesthawk** [sf-inc/sihywtcamd#19] [Ideas]
When playing Minecraft dungeons the spiders shoot webs at you, maybe you could try adding that as a mechanic. That doesn't sound like it would be an easy thing to do though, so maybe they could spawn a web on you when they attack instead or give you a slowness effect to simulate being webbed by a spider.

**Comment by charlyb01** [comment]
Spiders are not a big deal when it comes to kill them so your idea could really add some challenge. I like it

**Reply by deftesthawk** [reply]
I’m glad you like it, the spider swarm mechanic is super fun and thrilling, so more around that mechanic will definitely add to that

**Reply by charlyb01** [reply]
Wow, I didn't imagine it would make spider overpowered! Can't wait to let you try =)

**Reply by charlyb01** [reply]
[sihywtcamd-1.6-beta.zip](https://github.com/sf-inc/sihywtcamd/files/7025560/sihywtcamd-1.6-beta.zip)
Here is a beta version if you want to try and give feedbacks. I guess I need to tweak a few things like:
* range to launch a projectile (including a minimum distance to launch webs)
* time between two launches
* speed of the launch, I was thinking about adding a random percentage for this value
* what about baby spiders, do they have the ability, a different range, etc
* anything else you think of ;)

**Reply by deftesthawk** [reply]
That sounds awesome, maybe the baby spiders could give you slowness effect instead so you're not constantly stuck in webs.  don't have time to test at moment, I'll let you know if I manage to :)

**Reply by deftesthawk** [reply]
I tested out the spider and I love it, super awesome, it just works so well in combat

**Reply by charlyb01** [reply]
I'm glad you like it, I found it awesome too. I just need to tweak values I told you I think, if you have any suggestions

**Comment by charlyb01** [comment]
I've been working a bit more on it but sadly I won't have time anymore to continue for now. I may be able to code a bit sometime so I may release it before 1.18, but not so sure, I'm sorry. I'll try to send you beta if I add new things that I don't have time to test if you want

**Comment by deftesthawk** [comment]
No problem, I’m currently working on a modpack using forge now as it gives
me more options like editing/deleting recipes, adding stuff to the game,
questing mods and locking content behind ages.

**Comment by charlyb01** [comment]
Feel free to reopen discussion if needed =)
